src/bindgen/ir/function.rs

```diff
@@ -246,6 +258,10 @@ impl Source for Function {
                 write!(out, " {}({})", swift_name_macro, func.swift_name());
             }
 
+            if let Some(no_return_attr) = &config.function.no_return {
+                out.write_fmt(format_args!(" {} ", no_return_attr));
```

Comment:
Same here.

Also, for consistency, use `if let Some(ref ..) = config... {` rather than `if let Some(..) = &config... {`?

Reply:
(Also do we really need the trailing space?)